src/services/services.ts

```diff
@@ -2300,6 +2303,16 @@ export function createLanguageService(
         };
     }
 
+    function preparePasteEditsForFile(fileName: string, copiedTextRange: TextRange[]): boolean {
+        const sourceFile = syntaxTreeCache.getCurrentSourceFile(fileName);
+        bindSourceFile(sourceFile, getDefaultCompilerOptions());

```

Comment:
You are not suppose to bind the file on syntax server. Also the options used here are wrong as they should be `host.getCompilationSettings()`

Reply:
Without the binding, I don't get any `locals` or `symbol.exports` in the sourceFile. Is there a different way of doing this?

Reply:
If thats the case you would need this command on "partial semantic" or "semantic server" instead